src/pushsource/_impl/schema/staged-schema.yaml

```diff
@@ -104,6 +104,53 @@ definitions:
                 type:
                 - boolean
                 - "null"
+            release_notes:
+                $ref: "#/definitions/optional_string"
+
+            usage_instructions:
+                $ref: "#/definitions/optional_string"
+
+            recommended_instance_type:
+                $ref: "#/definitions/optional_string"
+
+            marketplace_entity_type:
+                $ref: "#/definitions/optional_string"
+
+            scanning_port:
+                type:
+                    - number
+                    - "null"
+                minimum: 0
+                maximum: 65353
+
+            user_name:
+                $ref: "#/definitions/optional_string"
+            version_title:
+                $ref: "#/definitions/optional_string"
+            security_groups:
+                type:
+                    - array
+                    - "null"
+                items:
+                    type: object
+
+                properties:
```

Comment:
I don't think this works, doesn't this part need to be indented by one more level so it's nested under "items"?  Otherwise the properties are being applied onto security_groups (the list), rather than the items within the list.

If in doubt - you can write a test which intentionally provides some invalid values under security_groups and check whether it really fails at these properties.

Reply:
did fixed this one to validate the security groups data properly